pkg/cmd/server/origin/asset.go

```diff
@@ -218,6 +220,20 @@ func (c *AssetConfig) addHandlers(mux *http.ServeMux) error {
 	}
 	mux.Handle(configPath, assets.GzipHandler(configHandler))
 
+	// Server version
+	kVersionInfo := kversion.Get()
+	oVersionInfo := oversion.Get()
+	versionInfo := assets.WebConsoleVersion{
+		KubernetesVersionInfo: kVersionInfo,
+		OpenShiftVersionInfo:  oVersionInfo,
+	}
+	versionHandler, err := assets.VersionHandler(versionInfo)
+	if err != nil {
+		return err
+	}
+	versionPath := path.Join(publicURL.Path, "version.js")
```

Comment:
All `config.js` entries are something you can change to customize some behavior, while version info is always generated by the master, not part of the set of configs an installation could change. Didn't seem to fit well. But I could buy having it in config to save a request, I think.